docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -2121,6 +2121,34 @@ rpc-ws-port="6174"
 The Websockets JSON-RPC listening port (TCP). The default is 8546. You must
 [expose ports appropriately](../../HowTo/Find-and-Connect/Configuring-Ports.md).
 
+### security-module
+
+```bash tab="Syntax"
+--security-module=<NAME>
+```
+
+```bash tab="Command Line"
+--security-module=hsm_plugin
+```
+
+```bash tab="Environment Variable"
+BESU_SECURITY_MODULE=hsm_plugin
+```
+
+```bash tab="Configuration File"
+security-module="hsm_plugin"
+```
+
+Name of the security module to use.
+
+Defaults to using the local private key file, and cannot be used with the
```

Comment:
do you know what behaviour actually exists - i.e. do we show an error, or are we just suggesting people should use them together?

Reply:
@rain-on we can't test this in Pliny per the thread about this so don't know what the actual behaviour is.